src/hooks/useTxDetails/useTxDetails.test.ts

```diff
@@ -23,3 +31,67 @@ describe('getSellTx', () => {
     expect(getSellTx(TradeTx)).toEqual(TradeTx.transfers[1])
   })
 })
+
+describe('getTransferByType', () => {
+  describe('TxType.Send', () => {
+    it('finds transfer with TxType.Send', () => {
+      const result = getTransferByType(EthSend, chainAdapters.TxType.Send)
+      const expected = EthSend.transfers[0]
+      expect(result).toEqual<chainAdapters.TxTransfer>(expected)
```

Comment:
Removed. I had added the type because I was reading documentation somewhere that passing the type ensured the compared values were also the same type.